src/com/company/Main.java

```diff
@@ -1,8 +1,119 @@
 package com.company;
 
+import java.time.LocalDate;
+import java.util.Arrays;
+
 public class Main {
+    public static void printIfLeapYear(int enteredYear) {
+        System.out.println(enteredYear + " год является високосным");
+    }
+
+    public static void printIfNotLeapYear(int enteredYear) {
+        System.out.println(enteredYear + " год не является високосным");
+    }
```

Comment:
лучше пометить как приватные, поскольку являются частью другого метода
```suggestion
    private static void printIfLeapYear(int enteredYear) {
        System.out.println(enteredYear + " год является високосным");
    }

    private static void printIfNotLeapYear(int enteredYear) {
        System.out.println(enteredYear + " год не является високосным");
    }
```